### ISSUE TITLE
Welcomer

### ISSUE THREAD

**Issue by bouleytator** [Repulser/bursting-cogs#14]
**The leave message isn't working. Even if it's enabled and has a message. Please help!**

**Comment by iTigerTone** [comment]
im having the same issue as well. can anyone help ?!

**Comment by Repulser** [comment]
Is the format set correctly?

**Comment by iTigerTone** [comment]
yes i believe so. It tells me that it's active but when someone leaves it doesnt post a message

________________________________

**Comment by ETLegacyy** [comment]
It doesnt work :/

**Comment by unseenmagik** [comment]
the issue i have found is when you use {2.name}
remove that and it should work fine. Mine has been ok after not using that

**Comment by iTigerTone** [comment]
i'll try that thank you

Sent from my iPhone

On Sep 7, 2017, at 06:51, unseenmagik <notifications@github.com<mailto:notifications@github.com>> wrote:


the issue i have found is when you use {2.name}
remove that and it should work fine. Mine has been ok after not using that

**Comment by rjm-codes** [comment]
O O F
